catalog-rest-service/src/main/resources/json/schema/metadataIngestion/databaseServiceMetadataPipeline.json

```diff
@@ -28,6 +28,24 @@
       },
       "additionalProperties": false
     },
+    "dbtCloudConfig": {
+      "title": "DBT Cloud Config Source",
+      "description": "DBT Catalog and Manifest HTTP path configuration.",
+      "type": "object",
+      "properties": {
+        "dbtCloudAuthToken": {
+          "description": "DBT cloud account authentication token",
```

Comment:
Could you please add `title` to those properties? they will show nicer in the UI